rules/autocmd.js

```diff
@@ -143,8 +145,8 @@ module.exports = {
 
   _autocmd_pattern: ($) => commaSep1(alias(/[^ \t\n,]+/, $.pattern)),
 
-  au_once: ($) => "++once",
-  au_nested: ($) => "++nested",
+  au_once: ($) => "++once ",
```

Comment:
Is this change necessary?

Reply:
No i had it for testing forgot to remove it.